recipes/app1.rb

```diff
@@ -18,23 +18,27 @@
 
 include_recipe 'osl-app::default'
 
-node.normal['users'] = %w(openid-staging openid-production)
+node.normal['users'] = %w(openid-staging openid-production) # cookstyle: disable ChefCorrectness/NodeNormal
```

Comment:
We probably should change this to the following:
```suggestion
node.default['users'] = %w(openid-staging openid-production) # cookstyle: disable ChefCorrectness/NodeNormal
```